create/redis.go

```diff
@@ -0,0 +1,92 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+type redisCmd struct {
+	Name            string                       `arg:"" default:"" help:"Name of the Redis instance. A random name is generated if omitted."`
+	Location        string                       `default:"nine-es34" help:"Location where the Redis instance is created."`
+	RedisVersion    storage.RedisVersion         `help:"Version specifies the Redis version."`
+	MemorySize      string                       `help:"MemorySize configures Redis to use a specified amount of memory for the data set."`
+	MaxMemoryPolicy storage.RedisMaxMemoryPolicy `help:"MaxMemoryPolicy specifies the exact behavior Redis follows when the maxmemory limit is reached."`
+	AllowedCidrs    []storage.IPv4CIDR           `help:"AllowedCIDRs specify the allowed IP addresses, connecting to the instance."`
```

Comment:
For some of these it would make sense to set the `placeholder:"something"` marker to give an example to the user what a valid arg value might look like in the help.